app/renderers/curation_concerns/attribute_renderer.rb

```diff
@@ -32,13 +32,13 @@ def render
       markup.html_safe
     end
 
-    private
+    def label
```

Comment:
Add a comment on what this method is for. Note that it could be overridden (so we don't make it private again)